src/test/versionbits_tests.cpp

```diff
@@ -302,17 +330,29 @@ static void check_computeblockversion(const Consensus::Params& params, Consensus
         nHeight += 1;
     }
 
-    nTime = nTimeout;
-    // FAILED is only triggered at the end of a period, so CBV should be setting
-    // the bit until the period transition.
-    for (uint32_t i = 0; i < params.nMinerConfirmationWindow - 1; i++) {
+    if (nTimeout != Consensus::BIP9Deployment::NO_TIMEOUT) {
+        // cannot reach NO_TIMEOUT a uint32_t
```

Comment:
µnit: grammar a weird

Reply:
I couldn't even fit all the words I needed a uint32_t.

Reply:
Replaced comment